lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -1,44 +1,52 @@
 # bgp_af.yaml
 ---
 _template:
-  config_get: 'show running bgp all'
   config_get_token: '/^router bgp <asnum>$/'
   config_get_token_append:
     - '/^vrf <vrf>$/'
     - '/^address-family <afi> <safi>$/'
   config_set: "router bgp <asnum>"
   config_set_append:
-    - 'vrf <vrf>'
-    - 'address-family <afi> <safi>'
+    - ' vrf <vrf>'
+    - ' address-family <afi> <safi>'
+  cli_nexus:
+    config_get: 'show running bgp all'
+  cli_ios_xr:
+    config_get: 'show running-config router bgp'
 
 additional_paths_install:
   kind: boolean
   config_get_token_append: '/^additional-paths install backup$/'
-  config_set_append: '<state> additional-paths install backup'
+  config_set_append: ' <state> additional-paths install backup'
   default_value: false
 
 additional_paths_receive:
   kind: boolean
   config_get_token_append: '/^additional-paths receive$/'
-  config_set_append: '<state> additional-paths receive'
+  config_set_append: ' <state> additional-paths receive'
   default_value: false
 
 additional_paths_selection:
   kind: string
-  config_get_token_append: '/^additional-paths selection route-map (.*)$/'
-  config_set_append: '<state> additional-paths selection route-map <route_map>'
+  cli_nexus:
+    config_get_token_append: '/^additional-paths selection route-map (.*)$/'
+    config_set_append: ' <state> additional-paths selection route-map <route_map>'
+  cli_ios_xr:
+    config_get_token_append: '/^additional-paths selection route-policy (.*)$/'
+    config_set_append: ' <state> additional-paths selection route-policy <route_policy>'
   default_value: ""
 
 additional_paths_send:
   kind: boolean
   config_get_token_append: '/^additional-paths send$/'
-  config_set_append: '<state> additional-paths send'
+  config_set_append: ' <state> additional-paths send'
   default_value: false
 
 advertise_l2vpn_evpn:
+  _exclude: [cli_ios_xr]
   kind: boolean
   config_get_token_append: '/^advertise l2vpn evpn$/'
-  config_set_append: '<state> advertise l2vpn evpn'
+  config_set_append: ' <state> advertise l2vpn evpn'
```

Comment:
After Charles explained the bug to me, I did a replace-all.  I consider this a latent bug and if it doesn't hurt Nexus, then adding a space may possibly eliminate future copy-paste recurrences of the bug.

I've removed the space from all nexus-only lines, but left it in all XR and XR/NX lines.